### PR TITLE
Integrate Regression Test with Jenkins

### PR DESCRIPTION
**Pull request by VoVAllen**
## Description
Please read benchmarks/scripts/README.md

## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR

### CODE REVIEW

File: benchmarks/scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Launch the whole regression test
+This will read the conf from the `benchmarks/task.json`. 
+```bash
+python run_reg_test.py
+```
+
+### Spec of task.json
+```json
+# Note the test will be run if the name specified below is a substring of the full test name.
```
Comment (classicsong) [inline benchmarks/scripts/README.md:12]: Substring is ambiguous, how about  a simplified Regular expression?

File: benchmarks/scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Regression Test Suite
+========================
+
+## Launch the whole regression test
+This will read the conf from the `benchmarks/task.json`. 
+```bash
+python run_reg_test.py
+```
+
+### Spec of task.json
```
Comment (classicsong) [inline benchmarks/scripts/README.md:10]: Do we have a concrete doc describe this task.json?

File: benchmarks/scripts/build_dgl_asv.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+. /opt/conda/etc/profile.d/conda.sh
+
+# Default building only with cpu
+DEVICE=${DGL_BENCH_DEVICE:-cpu}
+
+# build
+if [[ $DEVICE == "cpu" ]]; then
+    CMAKE_VARS=""
+else
+    CMAKE_VARS="-DUSE_CUDA=ON"
```
Comment (classicsong) [inline benchmarks/scripts/build_dgl_asv.sh:14]: Will we add TVM support in regression test? @jermainewang

File: benchmarks/scripts/ec2_script.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
+export INSTANCE_TYPE=$(curl -s http://169.254.169.254/latest/meta-data/instance-type)
```
Comment (classicsong) [inline benchmarks/scripts/ec2_script.sh:1]: This is a very bad thing to direct expose ip address in github.

File: benchmarks/scripts/launch_ec2.py [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+import time
+import boto3
+import argparse
+import os
+import textwrap
+import botocore
+
+
+def pass_env(extra_env_var):
+    env = "export INSTANCE_ID=$(curl -s http://169.254.169.254/latest/meta-data/instance-id) \n" + \
```
Comment (classicsong) [inline benchmarks/scripts/launch_ec2.py:11]: Why we need this?

File: benchmarks/scripts/ec2_script.sh (hunk 4 above)
Comment (VoVAllen) [reply benchmarks/scripts/ec2_script.sh]: This is a public feature of aws ec2 instance. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html

File: benchmarks/scripts/launch_ec2.py (hunk 5 above)
Comment (VoVAllen) [reply benchmarks/scripts/launch_ec2.py]: Removed

**Review by jermainewang — APPROVED** [review]

**Pull request merged**